shoop_tests/core/test_contacts.py

```diff
@@ -106,6 +106,44 @@ def test_person_contact_creating_from_user(regular_user):
     assert person.email == user.email
 
 
+@pytest.mark.django_db
+def test_person_name_init_by_name():
+    john = PersonContact(name="John Smith")
+    assert john.name == "John Smith"
+    assert john.first_name == "John"
+    assert john.last_name == "Smith"
+
+
+@pytest.mark.django_db
+def test_person_name_create_with_name():
+    john = PersonContact.objects.create(name="John Smith")
+    assert PersonContact.objects.get(pk=john.pk).name == "John Smith"
+    assert john.name == "John Smith"
+    assert john.first_name == "John"
+    assert john.last_name == "Smith"
```

Comment:
None of these test cases address the case where there are more than two space-separated words in the name...